plugins/Actions/DataTable/Filter/Actions.php

```diff
@@ -66,6 +67,14 @@ public function filter($table)
                     $label = $row->getColumn('label');
                     if ($url) {
                         $row->setMetadata('segmentValue', urlencode($url));
+
+                        if ($site && strpos($url, 'http://') === 0) {
+                            $host = parse_url($url, PHP_URL_HOST);
+
+                            if ($host && PageUrl::shouldUseHttpsHost($site->getId(), $host)) {
+                                $row->setMetadata('url', 'https://' . Common::mb_substr($url, 7 /* = strlen('http://') */));
```

Comment:
Is there a reason for using `mb_substr` here but normal `strtolower` above?

Reply:
will change that. even though it actually doesn't make a difference. See #16754